src/audio/mux/mux.c

```diff
@@ -200,7 +205,231 @@ static struct comp_dev *mux_new(const struct comp_driver *drv,
 	dev->state = COMP_STATE_READY;
 	return dev;
 }
+#else
+static int build_config(struct comp_data *cd, struct comp_dev *dev)
+{
+	dev->ipc_config.type = SOF_COMP_MUX;
+	cd->config.frame_format_deprecated = cd->md.base_cfg.audio_fmt.depth;
```

Comment:
One use of setting "frame_format_deprecated" still. This is not needed. This field has not been used since 23ae5d824ac180232835df355403a3a838b02fcc commit in 2020.